docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,2 @@
+
+### CONTRIBUTING
```

Comment:
this contributing.md can be removed, see #148

Reply:
done